portfolio.html

```diff
@@ -6,21 +6,25 @@
 		<title>My Portfolio</title>
 		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
 	</head>	
+	
 	<body>
+		
 		<nav class="nav_bar">
 			<div><label class="label_karen_schulz" for="nav_bar">Karen Schulz</label></div>
```

Comment:
I think you need to have 'label for="NavBar" before having a class. Or else have the class element in a seperate div or p tag

Reply:
changed the label issue.